src/main/draft/draftService.ts

```diff
@@ -35,6 +35,18 @@ export class DraftService {
     return client.find({ type: draftType, limit: limit }, userToken, deserializationFn, secrets)
   }
 
+  async readOne<T> (
```

Comment:
I think this should just be called `read`

Reply:
done